src/javascript/jsdoc.ts

```diff
@@ -74,6 +74,11 @@ function parsePolymerElement(tag: doctrine.Tag): Tag {
   return {tag: tag.title, type: null, name: tag.description, description: null};
 }
 
+// @polymerMixin [name]
+function parsePolymerMixin(tag: doctrine.Tag): Tag {
+  return {tag: tag.title, type: null, name: tag.description, description: null};
```

Comment:
How does this behave with annotations that include a description, like

```javascript
/**
 * @polymerMixin MyMixin A mixin that adds random yodeling through the web audio apis.
 */
const MyMixin = function(clazz) {...};
```

or even:

```javascript
/**
 * @polymerMixin A mixin that adds random yodeling through the web audio apis.
 */
const MyMixin = function(clazz) {...};
```